nbs/tutorial.ipynb

```diff
@@ -18,9 +18,7 @@
    "source": [
     "# Tutorial\n",
     "\n",
-    "> A step-by-step guide to using nbdev\n",
-    "\n",
-    "- image: images/card.png"
```

Comment:
Why was this removed?

Reply:
I noticed that it wasn't working: https://nbdev.fast.ai/tutorial.html and `card.png` is already used on the index page so I guessed that it wasn't intended here. What is this `-image: foo.png` syntax - it is a list? Is it a quarto extension to markdown?

Reply:
The `>` thing certainly works - it adds the description to the metadata of the notebook and rendered page.

The bullet points with YAML underneath is an alternative way of adding frontmatter. I'm not sure it works yet (it used to in nbdev1) but it's something I'll be getting to in the next day or 2 if not.

Reply:
Yes, I didn't delete the `>`. It's pushed to the next line in the git diff.

The bullet point thing definitely doesn't work. It's a bit confusing when things aren't doing anything (no errors, no output, although that should have been a clue in hindsight - that it didn't even output a string) and my bias is towards cleaning up... but if you plan to fix it I won't remove it here

Reply:
I think that bullet point thing used to work and then stopped working when we rolled back another PR supporting legacy front matter styles with markdown.  So good catch.

Seems like Jeremy is working on this right now though

Reply:
Created an issue to track: https://github.com/fastai/nbdev/issues/873